charms/kfp-api/src/charm.py

```diff
@@ -290,6 +290,8 @@ def _generate_environment(self) -> dict:
             "OBJECTSTORECONFIG_HOST": f"{object_storage['service']}.{object_storage['namespace']}",
             "OBJECTSTORECONFIG_PORT": str(object_storage["port"]),
             "OBJECTSTORECONFIG_REGION": "",
+            "V2_LAUNCHER_IMAGE": self.model.config["launcher-image"],
+            "V2_DRIVER_IMAGE": self.model.config["driver-image"],
```

Comment:
I see our defaults atm are `""`, so we're setting `V2_*_IMAGE=""` for the normal case.  Am I understanding this correctly that the upstream code interprets that as us wanting their default image?  I think it must if its passing tests.

Reply:
Yep, confirming this part was what took me most of the time implementing this yesterday. Upstream uses default image since it interprets empty string as a not defined environment variable.

Reply:
In any case, we 'll update the default value of the config so this shouldn't be an issue.